LucasTerra.lua

```diff
@@ -6990,4 +6990,58 @@ function pvpsafeworld()
 	return not table.find({'Chrona', 'Morta', 'Mortera', 'Calva', 'Calvera', 'Eldera', 'Dolera', 'Inferna'}, $worldname)
 end
 
+-- @name 	usedskill
+-- @desc 	Returns skill that's in use. If no skill is found, returns 'none'.
+-- @returns string
+function usedskill()
+	local skillTypes = {'sword', 'axe', 'club', 'distance', 'magic'}
+
+	local skill = 'none'
+	if $vocation == 'sorcerer' or $vocation == 'druid' then
+		skill = 'magic'
+	elseif $vocation == 'paladin' then
+		skill = 'distance'
+	else
+		if $axe > $sword and $axe > $club then
+			skill = 'axe'
+		elseif $sword > $axe and $sword > $club then
+			skill = 'sword'
+		else
+			skill = 'club'
+		end
+	end
+
+	return skill
+end
+
+-- @name 	usetrainer (credits to @Raphael)

```

Comment:
Can you move the `credits ...` part to the `desc` field? Having it here might break our parser one day.